internal/api/mfa.go

```diff
@@ -686,6 +682,36 @@ func (a *API) VerifyFactor(w http.ResponseWriter, r *http.Request) error {
 	metering.RecordLogin(string(models.MFACodeLoginAction), user.ID)
 
 	return sendJSON(w, http.StatusOK, token)
+}
+
+func (a *API) VerifyFactor(w http.ResponseWriter, r *http.Request) error {
+	ctx := r.Context()
+	factor := getFactor(ctx)
+	config := a.config
+
+	params := &VerifyFactorParams{}
+	if err := retrieveRequestParams(r, params); err != nil {
+		return err
+	}
+	if params.Code == "" {
```

Comment:
This is a refactor, moved the check out of the function since we're not considering WebAuthn for now.